### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by AmirSavand**
# Sidebar

Using a clean sidebar instead of the navbar.

## Tasks

- [x] Ability to close (minimize)
- [x] Auto close for mobile
- [x] Account section
- [x] Main nav links
- [x] Site selection

### CODE REVIEW

**Comment by AmirSavand** [comment]
@Alir3z4 we're blocked till backend sends logo of site.

File: app/header/header_view.html [hunk 1]
```diff
@@ -8,115 +8,89 @@
...
 </nav>
 
+<!-- Sidebar content -->
+<div class="sidebar-content" ng-class="{'sidebar-close': !set.sidebar}" ng-if="auth.isAuthenticated()">
+  <!-- Sites dropdown -->
+  <div class="dropdown dis-g">
+    <!-- Header -->
+    <a class="main dropdown-toggle t-line" data-toggle="dropdown" ng-click="set.sidebar = true">
+      <!-- Logo -->
+      <img ng-src="{{ user.getSites()[param.s].media.logo.thumbnail_48x48 }}">
```
Comment (Alir3z4) [inline app/header/header_view.html]: @AmirSavand what if their's no Logo ?

Comment (AmirSavand) [reply app/header/header_view.html]: I'll hide it.

**Comment by Alir3z4** [comment]
Alright, good

**Pull request merged**